test/validators.js

```diff
@@ -2493,6 +2493,19 @@ describe('Validators', () => {
       ],
     });
 
+    test({
+      validator: 'isPassportNumber',
+      args: ['MY'],
+      valid: [
+        'A00000000',
+        'H12345678',
```

Comment:
All looks good, but please add a test to catch `K`. 
Apart from that and fixing merge conflicts, this will be good to land.

Reply:
Added.